tensorflow-core/tensorflow-core-api/src/gen/java/org/tensorflow/op/core/ApproxTopK.java

```diff
@@ -0,0 +1,326 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Do we auto insert the copyright year when generating? If so, maybe it's worth updating the year to be "2018 - <current year>".

Reply:
Yep looks like that coming from us, I've updated it. Also I did the same change for the `Op` annotation processor (the one that generates the `*Ops.java` classes)